package.json

```diff
@@ -65,13 +71,15 @@
     "package-mac": "electron-builder build --mac -c.extraMetadata.main=build/electron-main.js --publish never",
     "package-linux": "electron-builder build --linux -c.extraMetadata.main=build/electron-main.js --publish never",
     "package-win": "electron-builder build --win -c.extraMetadata.main=build/electron-main.js --publish never",
-    "eslint": "eslint {electron,src}/**/*.{js,jsx,json,css,scss,md}",
+    "eslint": "eslint './{src,electron}/**/*.{js,jsx,json,css,scss,md}' --fix",
     "prettier": "prettier --write '{electron,src}/**/*.{js,jsx,json,css,scss,md}'",
     "test": "react-scripts test",
     "test-react": "react-scripts test",
     "test-electron": "jest --detectOpenHandles electron/**/*.js --watch --verbose",
     "cucumber": "./node_modules/.bin/cucumber-js src/react --no-strict -r src/react/utils/cucumber --require-module @babel/register --require-module @babel/polyfill",
     "cucumber-only": "./node_modules/.bin/cucumber-js src/react/features --tags @only -r src/react/utils/cucumber --require-module @babel/register --require-module @babel/polyfill",
+    "storybook": "start-storybook -p 9009 -s public",
+    "build-storybook": "build-storybook -s public",
```

Comment:
Do you want to host this statically somewhere? (as that what I understood is the purpose of build-storybook...)

I think it's enough to have it only in the codebase by now...

Reply:
No need for that sofar.